service/src/main/kotlin/no/nav/su/se/bakover/service/kontrollsamtale/KontrollsamtaleService.kt

```diff
@@ -99,10 +248,28 @@ class KontrollsamtaleServiceImpl(
     }
 }
 
+sealed class KunneIkkeSetteNyDatoForKontrollsamtale {
```

Comment:
```suggestion
sealed interface KunneIkkeSetteNyDatoForKontrollsamtale {
```

Skal lage en PR senere som bytter de fleste sealed class til interface. Får litt fler arvemuligheter da. Bør kunne hjelpe oss med å forenkle noen av disse feil-typene